releases/release-1.31/release-notes/maps/pr-125163-map.yaml

```
@@ -0,0 +1,6 @@
+pr: 125163
+releasenote:
+  text: 'DRA: ACTION REQUIRED: The Dynamic Resource Allocation (DRA) driver's DaemonSet must be deployed
```

```suggestion
  text: 'ACTION REQUIRED: The Dynamic Resource Allocation (DRA) driver's DaemonSet must be deployed
```
I think people look for `^ACTION REQUIRED:`; a DRA prefix might be unhelpful.